docs/source/en/model_doc/helium.md

```diff
@@ -107,16 +107,16 @@ Tips:
   
 ## Usage tips
 
-`Helium` can be found on the [Huggingface Hub](https://huggingface.co/collections/kyutai/helium-1-preview)
+`Helium` can be found on the [Huggingface Hub](https://huggingface.co/models?other=helium)
 
 In the following, we demonstrate how to use `helium-1-preview` for the inference. 
 
 ```python
 >>> from transformers import AutoModelForCausalLM, AutoTokenizer
 >>> device = "cuda" # the device to load the model onto
 
->>> model = AutoModelForCausalLM.from_pretrained("helium-1-preview", device_map="auto")
->>> tokenizer = AutoTokenizer.from_pretrained("helium-1-preview")
+>>> model = AutoModelForCausalLM.from_pretrained("kyutai/helium-1-preview-2b", device_map="auto")
+>>> tokenizer = AutoTokenizer.from_pretrained("kyutai/helium-1-preview-2b")
```

Comment:
@ArthurZucker let me know if that's the wrong checkpoint -> you're using

```py
>>> text = tokenizer.apply_chat_template(messages, tokenize=False, add_generation_prompt=True)
```

below, but this checkpoint does not have a chat template.

Reply:
yep thanks